layouts/partials/footer.html

```diff
@@ -7,6 +7,10 @@
     {{ now.Year }}
     {{ with .Site.Params.author }} {{ . }} {{ end }}
     ·
+    {{ if (and .Site.Params.license) }}
+      {{ i18n "licensed_under" }} {{ .Site.Params.license }}
```

Comment:
Perhaps this should be filtered with `safeHTML`:

```
{{ .Site.Params.license | safeHTML }}
```

https://gohugo.io/functions/safehtml/

Reply:
That's a good one, that would allow us to include a link to the license. Will update this one.